src/app/pages/admin/admin.controller.js

```diff
@@ -31,14 +31,14 @@ export default class AdminController {
         this.sharedAdminFactory.setUserData(id);
     }
 
-    resetPsw(userId, email) {
-        this.adminService.resetUserPassword({ uid: userId, newPassword: email });
+    resetPsw(uid, email) {
+        this.adminService.resetUserPassword({ uid: uid, newPassword: email });
```

Comment:
It is unnecessary -   uid: uid,   you may write:
                    this.adminService.resetUserPassword({ uid, newPassword: email });

Reply:
Fixed